artemis-core-client/src/main/java/org/apache/activemq/artemis/api/core/Message.java

```diff
@@ -186,10 +186,12 @@ default Message setRoutingType(RoutingType routingType) {
       return this;
    }
 
+   @Deprecated
```

Comment:
Why deprecate this?


AMQP has some properties within the Spec that could be used. So this could be Protocol Specific. Hence why we had this.

Reply:
No need ill remove